dataedit/metadata/widget.py

```diff
@@ -0,0 +1,247 @@
+import re
+
+from django.utils.safestring import mark_safe
+
+from dataedit.metadata import METADATA_HIDDEN_FIELDS
+
+LICENSE_KEY = 'license'
+COLUMNS_KEY = 'fields'
+
+
+class MetaDataWidget:
```

Comment:
This create either an html normal view or an html form view of a nested structure which can contain dict, list and strings